test/fixtures/for/nested.js

```diff
@@ -1,9 +1,9 @@
 let x
-for i from 0 til 10:
-  for j from 0 til 10:
+for let i=0; i<10; i++:
```

Comment:
in tests like this (many above) let's instead translate to

```
for idx i in Array(10):
```

which should probably be the idiomatic approach.